set.mm

```diff
@@ -510554,6 +510819,1209 @@ commutative monoid (=vectors) together with a semiring (=scalars) and a
+         by Thierry Arnoux, 4-May-2025.) $)
+      erlcl2 $p |- ( ph -> V e. ( B X. S ) ) $=
+        ( vt c1st cfv c2nd co wa eqid va vb cxp wcel cv cmulr csg c0g wceq wrex
+        wbr cerl copab erlval eqtrid simpl fveq2d oveq12d oveq2d eqeq1d rexbidv
+        wb simpr adantl brab2d mpbid simplrd ) AFBEUCZUDZGVHUDZNUEZFOPZGQPZDUFP
+        ZRZGOPZFQPZVNRZDUGPZRZVNRZDUHPZUIZNEUJZAFGCUKVIVJSWDSMAVKUAUEZOPZUBUEZQ
+        PZVNRZWGOPZWEQPZVNRZVSRZVNRZWBUIZNEUJZWDUAUBFGCVHVHACDEULRWEVHUDWGVHUDS
+        WPSUAUBUMZJANBWQDEVNVSHVHWBUAUBIWBTVNTVSTVHTWQTKLUNUOWEFUIZWGGUIZSZWPWD
+        VBAWTWOWCNEWTWNWAWBWTWMVTVKVNWTWIVOWLVRVSWTWFVLWHVMVNWTWEFOWRWSUPZUQWTW
+        GGQWRWSVCZUQURWTWJVPWKVQVNWTWGGOXBUQWTWEFQXAUQURURUSUTVAVDVEVFVG $.
+    $}
+
+    ${
+      erldi.1 $e |- .0. = ( 0g ` R ) $.
+      erldi.2 $e |- .x. = ( .r ` R ) $.
+      erldi.3 $e |- .- = ( -g ` R ) $.
+      ${
+        $d .- a b $.  $d .0. a b $.  $d .x. a b t $.  $d B a b t $.  $d B r $.
+        $d B s $.  $d R a b t $.  $d S a b t $.  $d S r $.  $d S s $.
+        $d U a b t $.  $d U r $.  $d U s $.  $d V a b t $.  $d a ph s $.
+        $d b ph $.  $d ph r $.
+        erldi.4 $e |- ( ph -> U .~ V ) $.
+        $( Main property of the ring localization equivalence relation.
+           (Contributed by Thierry Arnoux, 4-May-2025.) $)
+        erldi $p |- ( ph -> E. t e. S ( t .x. ( ( ( 1st ` U ) .x. ( 2nd ` V ) )
+                              .- ( ( 1st ` V ) .x. ( 2nd ` U ) ) ) ) = .0. ) $=
+          ( va vb cxp wcel wa cv c1st cfv c2nd co wceq wrex wbr cerl copab eqid
+          erlval eqtrid simpl fveq2d simpr oveq12d oveq2d eqeq1d rexbidv adantl
+          wb brab2d mpbid simprd ) AHCFUCZUDJVKUDUEZBUFZHUGUHZJUIUHZGUJZJUGUHZH
+          UIUHZGUJZIUJZGUJZLUKZBFULZAHJDUMVLWCUETAVMUAUFZUGUHZUBUFZUIUHZGUJZWFU
+          GUHZWDUIUHZGUJZIUJZGUJZLUKZBFULZWCUAUBHJDVKVKADEFUNUJWDVKUDWFVKUDUEWO
+          UEUAUBUOZNABCWPEFGIKVKLUAUBMQRSVKUPWPUPOPUQURWDHUKZWFJUKZUEZWOWCVGAWS
+          WNWBBFWSWMWALWSWLVTVMGWSWHVPWKVSIWSWEVNWGVOGWSWDHUGWQWRUSZUTWSWFJUIWQ
+          WRVAZUTVBWSWIVQWJVRGWSWFJUGXAUTWSWDHUIWTUTVBVBVCVDVEVFVHVIVJ $.
+      $}
+
+      $d .- a b t $.  $d .0. a b t $.  $d .x. a b t $.  $d B a b t $.
+      $d B r $.  $d B s $.  $d E a b t $.  $d F a b t $.  $d G a b t $.
+      $d H a b t $.  $d R a b t $.  $d S a b t $.  $d S r $.  $d S s $.
+      $d T t $.  $d U a b t $.  $d U r $.  $d U s $.  $d V a b t $.
+      $d a ph s $.  $d b ph t $.  $d ph r $.
+      erlbrd.u $e |- ( ph -> U = <. E , G >. ) $.
+      erlbrd.v $e |- ( ph -> V = <. F , H >. ) $.
+      erlbrd.e $e |- ( ph -> E e. B ) $.
+      erlbrd.f $e |- ( ph -> F e. B ) $.
+      erlbrd.g $e |- ( ph -> G e. S ) $.
+      erlbrd.h $e |- ( ph -> H e. S ) $.
+      erlbrd.1 $e |- ( ph -> T e. S ) $.
+      erlbrd.2 $e |- ( ph -> ( T .x. ( ( E .x. H ) .- ( F .x. G ) ) ) = .0. )
+        $.
+      $( Deduce the ring localization equivalence relation.  (Contributed by
```

Comment:
Again, maybe some details? Like
```
If for some ` T e. S ` we have ` T x. ( E x. H - F x. G ) = 0 `, then pairs ` <. E , G >. `
and ` <. F , H >. ` are equivalent under the localization relation.
```

Reply:
Fixed in https://github.com/metamath/set.mm/pull/4810/commits/f635662cfffd84458584f2c4a905b88f32b45a51